src/couch/src/couch_httpd_auth.erl

```diff
@@ -33,7 +33,7 @@
 
 -export([jwt_authentication_handler/1]).
 
--import(couch_httpd, [header_value/2, send_json/2,send_json/4, send_method_not_allowed/2]).
+-import(couch_httpd, [header_value/2, send_json/2, send_json/4, send_method_not_allowed/2, maybe_decompress/2]).
```

Comment:
why import the function if you're calling it with the module name below anyway?

Reply:
Good call. Done in a7c8cee76